--- a/scripts/py2dsc
+++ b/scripts/py2dsc
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 USAGE = """\
 usage: py2dsc [options] distfile
    or: py2dsc --help
--- a/scripts/py2dsc-deb
+++ b/scripts/py2dsc-deb
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 USAGE = """\
 usage: py2dsc-deb [options] distfile
    or: py2dsc-deb --help
--- a/scripts/pypi-download
+++ b/scripts/pypi-download
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 import sys
 from optparse import OptionParser
 from stdeb.downloader import myprint, get_source_tarball
--- a/scripts/pypi-install
+++ b/scripts/pypi-install
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 import sys, os, shutil
 import subprocess
 import argparse
